cdap-docs/developers-manual/source/building-blocks/core.rst

```diff
@@ -18,15 +18,15 @@ low-level APIs.
 .. rubric:: Application Abstraction
 
 Applications hide low-level details of individual programming paradigms and runtimes,
-while providing access to many useful and powerful services provided by CDAP such as 
+while providing access to many useful and powerful services provided by CDAP such as
 distributed transactions, service discovery, and the ability to dynamically scale
 processing units.
 
 Applications are abstracted away from the platform that runs the application. When you
 deploy and run the application into a specific installation of CDAP, the appropriate
 implementations of all services and program runtimes are injected by CDAP; the application
 does not need to change based on the environment. This allows you develop applications in
-one environment—such as on your laptop using a standalone CDAP for testing—and then
+one environment—such as on your laptop using a CDAP Sandbox for testing—and then
```

Comment:
We don't talk (yet) about "Cloud Sandbox", so I'm thinking this should be (for now) "CDAP Local Sandbox". Re-wrote sentence to correct a number of errors.